test/simapp/simd/cmd/testnet.go

```diff
@@ -78,7 +81,7 @@ Example:
 
 			return InitTestnet(
 				clientCtx, cmd, config, mbm, genBalIterator, outputDir, chainID, minGasPrices,
-				nodeDirPrefix, nodeDaemonHome, startingIPAddress, keyringBackend, algo, numValidators,
+				nodeDirPrefix, nodeDaemonHome, startingIPAddress, keyringBackend, algo, numValidators, clientCtx.TxConfig.SigningContext().ValidatorAddressCodec(),
```

Comment:
(https://github.com/cosmos/cosmos-sdk/pull/17098) add addresscodec for GenAppStateFromConfig func